examples/python/fix_connect_example.py

```diff
@@ -0,0 +1,108 @@
+import time
+import quickfix
+import quickfix44
+
+__SOH__ = chr(1)
```

Comment:
Any reason you use the __X__ format for your global? Also you only use this in one place so could it be defined within function scope?

Reply:
Made the changes